node/src/main.rs

```diff
@@ -317,6 +317,11 @@ async fn main() {
                     network_store.block_store(),
                     &logger_factory,
                 );
+
+                // Start a task runner
+                let mut job_runner = graph::util::jobs::Runner::new(&logger);
+                register_store_jobs(&mut job_runner, network_store.clone());
+                graph::spawn_blocking(job_runner.start());
```

Comment:
The use of `graph::spawn_blocking` should be avoided. If the runner is for futures, and therefore should not be blocking, then this should be just `spawn`.

Reply:
This is intentional: these jobs will all (or almost all) interact with the database. Just throwing everything wholesale into the blocking pool makes it easier to write those jobs.

Reply:
I see so even though the vacuum uses `with_conn` you don't want to promise they all will.